app/views/hq/partners/edit.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1> Edit Partner with Osra Number:- <%= @partner.osra_num %> </h1>
```

Comment:
":-" is uncommon - something special? I can see there being no punctuation at all here. What about substituting or adding `@partner.name`?